server/lib/chat/chat-service.ts

```diff
@@ -161,7 +181,10 @@ export default class ChatService {
     // is allowed in some cases (e.g. during account creation)
     const userSockets = this.userSocketsManager.getById(userInfo.id)
     if (userSockets) {
-      userSockets.subscribe(getChannelPath(channelId))
+      userSockets.subscribe<ChatInitActiveUsersEvent>(getChannelPath(channelId), () => ({
+        action: 'initActiveUsers',
+        activeUserIds: this.state.channels.get(channelId)!.toArray(),
```

Comment:
For safety can you please make this handle the undefined case instead of doing the non-null coercion? `this.state.channels.get(channelId)?.toArray() ?? []`